src/activities/content/ContentWebLinkEntity.js

```diff
@@ -61,6 +61,18 @@ export class ContentWebLinkEntity extends ContentWorkingCopyEntity {
 		return this._entity && this._entity.properties && this._entity.properties.url;
 	}
 
+	type() {
+		if (this._entity) {
+			if (this._entity.hasClass(Classes.webLink.googleDriveLink)) {
+				return Classes.webLink.googleDriveLink;
+			} else if (this._entity.hasClass(Classes.webLink.oneDriveLink)) {
+				return Classes.webLink.oneDriveLink;
+			}
+		}
+
+		return 'default';
```

Comment:
I wonder if we should instead move [this](https://github.com/BrightspaceHypermediaComponents/activities/pull/2385/files#diff-af9a9f9b02aff01e8acd70627d7cfecb271c14f214802866805ccf794acce65dR1) and [this](https://github.com/BrightspaceHypermediaComponents/activities/pull/2386/files#diff-af9a9f9b02aff01e8acd70627d7cfecb271c14f214802866805ccf794acce65dR8) into `ContentWebLinkEntity` (similar to files). That way we don't have multiple lists keeping track of the same thing and having to match them; we can just consolidate it here.

We probably also just need one list rather than separating them into WebLinkTypes and CloudDriveLinks.